api/providers/urls.py

```diff
@@ -8,9 +8,16 @@
     url(r'^preprints/$', views.PreprintProviderList.as_view(), name=views.PreprintProviderList.view_name),
     url(r'^preprints/(?P<provider_id>\w+)/$', views.PreprintProviderDetail.as_view(), name=views.PreprintProviderDetail.view_name),
     url(r'^preprints/(?P<provider_id>\w+)/licenses/$', views.PreprintProviderLicenseList.as_view(), name=views.PreprintProviderLicenseList.view_name),
-    url(r'^preprints/(?P<provider_id>\w+)/submissions/$', views.PreprintProviderPreprintList.as_view(), name=views.PreprintProviderPreprintList.view_name),
+    url(r'^preprints/(?P<provider_id>\w+)/preprints/$', views.PreprintProviderPreprintList.as_view(), name=views.PreprintProviderPreprintList.view_name),
     url(r'^preprints/(?P<provider_id>\w+)/taxonomies/$', views.PreprintProviderTaxonomies.as_view(), name=views.PreprintProviderTaxonomies.view_name),
     url(r'^preprints/(?P<provider_id>\w+)/taxonomies/highlighted/$', views.PreprintProviderHighlightedSubjectList.as_view(), name=views.PreprintProviderHighlightedSubjectList.view_name),
     url(r'^preprints/(?P<provider_id>\w+)/moderators/$', views.PreprintProviderModeratorsList.as_view(), name=views.PreprintProviderModeratorsList.view_name),
     url(r'^preprints/(?P<provider_id>\w+)/moderators/(?P<moderator_id>\w+)/$', views.PreprintProviderModeratorsDetail.as_view(), name=views.PreprintProviderModeratorsDetail.view_name),
+
+    url(r'^collections/$', views.CollectionProviderList.as_view(), name=views.CollectionProviderList.view_name),
+    url(r'^collections/(?P<provider_id>\w+)/$', views.CollectionProviderDetail.as_view(), name=views.CollectionProviderDetail.view_name),
+    url(r'^collections/(?P<provider_id>\w+)/licenses/$', views.CollectionProviderLicenseList.as_view(), name=views.CollectionProviderLicenseList.view_name),
+    #url(r'^collections/(?P<provider_id>\w+)/submissions/$', views.CollectionProviderSubmissionList.as_view(), name=views.CollectionProviderSubmissionList.view_name),
```

Comment:
coming soon or extra?

Reply:
There's a `# TODO: [IN-153]` over the commented-out class stub, I just didn't put that here too.